src/components/Global/Header/index.tsx

```diff
@@ -115,6 +115,37 @@ const MenuToggle = ({ toggle, isOpen }: { toggle: () => void; isOpen: boolean })
     )
 }
 
+const MenuLink = ({ route, title, isBeta = false }: { route: string; title: string; isBeta?: boolean }) => {
```

Comment:
thought: in the future we might also want some other properties (e.g. isNew or hasNotification or similar)